src/maplibre_helpers.ts

```diff
@@ -171,6 +178,23 @@ export function emptyGeojson(): FeatureCollection {
   };
 }
 
+export function pointFeature(pt: Position): FeatureWithProps<Point> {
+  return {
+    type: "Feature",
+    properties: {},
+    geometry: {
+      type: "Point",
+      coordinates: trimPrecision(pt),
+    },
+  };
+}
+
+// Per https://datatracker.ietf.org/doc/html/rfc7946#section-11.2, 6 decimal
+// places (10cm) is plenty of precision
+export function trimPrecision(pt: Position): Position {
```

Comment:
Can we call this `setPrecision`, `trimPrecision` is a bit confusing because we're either trimming the numbers to a precision, or (which seems naturally) we're setting the precision of the numbers

Reply:
Done! Yeah, `trimPrecision` sounds like it ought to take a second parameter for the specified level